packages/helm-charts/blockscout/templates/blockscout-web.deployment.yaml

```diff
@@ -98,6 +98,10 @@ spec:
           value: {{ .Values.blockscout.networkID | quote }}
         - name: APPS_MENU
           value: {{ .Values.blockscout.web.appsMenu.enabled | quote }}
+        - name: DISABLE_STATS
+          value: {{ not .Values.blockscout.web.stats.enabled | quote }}
```

Comment:
I would recommend avoiding mixing "enable" and "disable" semantics and stick with "enable" (so that true/false reads naturally without having to do disable + false = enable mental logic deduction)

Reply:
I agree. The thing is that in Blockscout's codebase it seems to be standard to use `DISABLE_` variables, while in helm charts we decided to stick to `enabled` everywhere.

Reply:
Makes sense then, unfortunately. :)